synapse/models/economic.py

```diff
@@ -448,9 +488,6 @@ def getModelDefs(self):
                     ('iban', ('econ:bank:iban', {}), {
                         'doc': 'The IBAN for the account.'}),
 
-                    ('contact', ('ps:contact', {}), {
```

Comment:
Same comment as for crypto:currency:address, we're deprecating one prop and replacing it with an inherited pro with the same name. Is this ok?